pkg/store/gcworker/BUILD.bazel

```diff
@@ -51,7 +51,7 @@ go_test(
     embed = [":gcworker"],
     flaky = True,
     race = "on",
-    shard_count = 29,
```

Comment:
Just curious, what does it mean?

Reply:
I don't know either, CI told me that I must update this..